Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -55,12 +60,20 @@ void LoadBankFromDiskTask::loadPulseTimes(::NeXus::File &file) {
   } catch (::NeXus::Exception &) {
     // Field not found error is most likely.
     // Use the "proton_charge" das logs.
-    thisBankPulseTimes = m_loader.alg->m_allBanksPulseTimes;
+    thisBankPulseTimes = m_loader.m_loadAlgorithm->m_allBanksPulseTimes;
     return;
   }
   std::string thisStartTime;
   size_t thisNumPulses = 0;
-  file.getAttr("offset", thisStartTime);
+  if (file.hasAttr("offset"))
+    file.getAttr("offset", thisStartTime);
+  else if (file.hasAttr("start"))
+    file.getAttr("start", thisStartTime);
+  else {
```

Comment:
Same as code in `BankPulseTimes.cpp`, minor point - consider refactoring. but also needs same comment.